integrationTests/singleShard/block/executingMiniblocksSc_test.go

```diff
@@ -438,20 +429,19 @@ func TestProcessesJoinGame100PlayersMultipleTimesRewardAndEndgameInMultipleRound
 	idxProposers[1] = 2
 	idxProposers[2] = 4
 
+	deployScTx(nodes, idxProposer, string(scCode))
+	time.Sleep(stepDelay)
 	for i := 0; i < nrRoundsToPropagateMultiShard; i++ {
-		time.Sleep(time.Second)
-		deployScTx(nodes, idxProposer, string(scCode))
-		time.Sleep(time.Second)
 		proposeBlock(nodes, idxProposers, round)
-		time.Sleep(time.Second)
```

Comment:
This deletion should be reverted. Let the sync nodes receive the header and body

Reply:
proposeBlock does have inside a time.sleep(stepDelay). that should be enough.